frontend/src/components/Catalogue.jsx

```diff
@@ -42,7 +42,7 @@ const Catalogue = () => {
 	useEffect(() => {
 		async function fetchCatalogue() {
 			try {
-				const response = await http.get("/catalogues/1");
```

Comment:
undo this change. This is part of a frontend audit and represents a miscommuncication between ourselves and the team. Users should be allowed more than one catalogue, it is not safe to assume that there will always be one catalogue. Good news is, we can pretty much just copy/paste the Site-making page and make it a catalouge making page.